master/buildbot/status/buildstep.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, parent, master, step_number, step_type):
         self.hidden = False
         self.logs = []
         self.urls = {}
+        self.artifactUrls = {}
```

Comment:
I'd rather it stay with ArtifactUrls, to clarify that it does contain URL links to the artifacts' actual locations. The less vague the variable name, the better.